### PR TITLE
chore(bot): update adaptive card notification sample

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Update the adaptive card notification sample

### CODE REVIEW

File: adaptive-card-notification/README.md [hunk 1]
```diff
@@ -41,8 +41,16 @@ Adaptive Card Notification provides an easy way to send notification in Teams. T
 ![factset](./images/factset.jpg)
 1. Send a POST request to `<endpoint>/api/list-notification`, you will receive a list adaptive card.
 ![list](./images/list.jpg)
-1. Update the `userId` and `userName` to the user who you want to mentioned in the file [`bot/src/adaptiveCards/notification-mention.data.json`](bot/src/adaptiveCards/notification-mention.data.json). Send a POST request to `<endpoint>/api/mention-notification`, you will receive a mention adaptive card.
-![mention](./images/mention.jpg)
+1. Update the `userId` and `userName` to the user who you want to mentioned in the file [bot/src/mentionNotificationHttpTrigger.ts](bot/src/mentionNotificationHttpTrigger.ts). Send a POST request to `<endpoint>/api/mention-notification`, you will receive a mention adaptive card.
```
Comment (dooriya) [inline adaptive-card-notification/README.md:44]: to mentioned -> to mention

Comment (dooriya) [inline adaptive-card-notification/README.md:44]: to -> via

Comment (dooriya) [inline adaptive-card-notification/README.md:44]: `mention adaptive card` sounds a little bit weird. How a bout `an adaptive card message that mentioned someone`?

File: adaptive-card-notification/bot/src/cardModels.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * Columnset sample data model bound to the columnset card template.
+ */
+export interface ColumnsetData extends CardData {
+    data: { property1: string, property2: string, property3: string }[];
+}
+
+/**
+ * Factset sample data model bound to the factset card template.
```
Comment (dooriya) [inline adaptive-card-notification/bot/src/cardModels.ts:19]: We can also add a reference link to the `FactSet` schema of Adaptative Card to help them better understand the schema if they'd like to have some customization. And the comment could be:
```
/**
 * Adaptive card data model that bound to the FactSet template in `adaptiveCards/notification-factset.json`.
 * For more details of the FacSet schema, you can refer to https://adaptivecards.io/explorer/FactSet.html
 */
```

Same for
- ColumnSet: https://adaptivecards.io/explorer/ColumnSet.html
- mention user: https://docs.microsoft.com/microsoftteams/platform/task-modules-and-cards/cards/cards-format?tabs=adaptive-md%2Cconnector-html#mention-support-within-adaptive-cards

File: adaptive-card-notification/README.md (hunk 1 above)
Comment (XiaofuHuang) [reply adaptive-card-notification/README.md]: Thx!

Comment (XiaofuHuang) [reply adaptive-card-notification/README.md]: got it!

File: adaptive-card-notification/bot/src/cardModels.ts (hunk 2 above)
Comment (XiaofuHuang) [reply adaptive-card-notification/bot/src/cardModels.ts]: Sure, added the comments

File: adaptive-card-notification/bot/src/cardModels.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * For more details of the FactSet schema, you can refer to https://adaptivecards.io/explorer/FactSet.html
+ */
+export interface FactsetData extends CardData {
+    factSet: { property1: string, property2: string, property3: string };
+}
+
+/**
+ * Adaptive card data model that bound to the List template in `adaptiveCards/notification-list.json`.
+ * For more details of the Container schema, you can refer to https://adaptivecards.io/explorer/Container.html
+ */
```
Comment (dooriya) [inline adaptive-card-notification/bot/src/cardModels.ts:31]: List is actually a degraded sample for ColumnSet, let's still refer to the ColumnSet schema link

Comment (XiaofuHuang) [reply adaptive-card-notification/bot/src/cardModels.ts:31]: Got it, updated the link.

**Review by dooriya — APPROVED** [review]

**Pull request merged**